relayer/create_ibc_channel.go

```diff
@@ -0,0 +1,111 @@
+package relayer
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	retry "github.com/avast/retry-go"
+	"github.com/dymensionxyz/roller/cmd/consts"
+	"github.com/dymensionxyz/roller/cmd/utils"
+)
+
+// Creates an IBC channel between the hub and the client, and return the source channel ID.
+func (r *Relayer) CreateIBCChannel(logFileOption utils.CommandOption) (ConnectionChannels, error) {
+	//TODO: add support for --override flag
+	createClientsCmd := r.getCreateClientsCmd()
+	fmt.Println("💈 Creating clients...")
+	if err := utils.ExecBashCmd(createClientsCmd, logFileOption); err != nil {
```

Comment:
Using exechbashcmd here can cause issues due to its propensity to crash, and the vague 'exit status 1' error message provides little insight. Consider modifying exechbashcmd to return stderr upon error or alternatively, switch to using ExecBashCmdWithOSOutput, which already implements this feature

Reply:
I don't want it to print to stdout, just to the logger